src/components/mobile-nav-popover/mobile-nav-popover.html

```diff
@@ -7,8 +7,8 @@
     <ion-icon name="close-circle-outline" (click)="closeNav()" class="close"></ion-icon>
   </div>
   <ng-container *ngIf="regularNavShow">
-    <button ion-button color="dark" disabled>
-      User Name
+    <button ion-button color="dark" disabled class="user-name">
+      {{this.userName}}
```

Comment:
```suggestion
      {{userName}}
```

Reply:
@dgonca you don't need to include ```this``` inside the double brackets. I am trying out the commit suggestion feature.... don't actually commit this.